### ISSUE TITLE
Crazy idea to make new app...

### ISSUE THREAD

**Issue by Meggin** [bart-on-time/bart-on-time#54]
I was thinking we could do something similar like this app for shuttle buses, where there's a schedule and live data.

It could be a very useful app when you can read traffic, and understand the actual arrival time versus the scheduled time, you location, and closest shuttle stops at any given time based on traffic.

Yeah, it would be awesome!

**Comment by taehyunlim** [comment]
Yes, we can definitely take this and make it more robust and bigger. I'll be really happy to work on it more just as a side project.
